config/environments/production.rb

```diff
@@ -81,6 +81,22 @@
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  config.lograge.enabled = true
+
+  config.lograge.custom_options = lambda do |event|
+    {}.tap do |h|
+      h[:host] = event.payload[:host]
+      h[:user_id] = event.payload[:user_id]
+      h[:user_email] = event.payload[:user_email]
+      h[:user_organisation_slug] = event.payload[:user_organisation_slug]
+      h[:request_id] = event.payload[:request_id]
+      h[:user_id] = event.payload[:user_id]
```

Comment:
I don't think we can log these in the API:
```
      h[:user_id] = event.payload[:user_id]
      h[:user_email] = event.payload[:user_email]
      h[:user_organisation_slug] = event.payload[:user_organisation_slug]
      h[:request_id] = event.payload[:request_id]
      h[:user_id] = event.payload[:user_id]
```
They exist in the admin and we could use the request id to link the two but otherwise they will always be blank?